### PR TITLE
Generic templates

### PR DESCRIPTION
**Pull request by drkameleon**
Added CODE OF CONDUCT file + ISSUE templates + PR templates.

In relation to issue #4

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at yaniszaf@gmail.com. All
```
Comment (Martinsos) [inline CODE_OF_CONDUCT.md:58]:
```suggestion
reported by contacting the project team at sosic.martin+wasp@gmail.com. All
```

**Review by Martinsos — APPROVED** [review]
Yay this will be our first external contribution :D!!
Thanks, these are all useful, especially the code of conduct, we were planning to do it but wanted to take some time to research it properly and choose the best option. Contributor Covenant looks cool and widespread, so great, let's do that!
I made one suggestion, to update the email -> please accept that and I will merge :).

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (drkameleon) [reply CODE_OF_CONDUCT.md:58]: Well, I guess I forgot to change that... haha!

Done. :)

Comment (Martinsos) [reply CODE_OF_CONDUCT.md]: Or maybe you wanted to be our CoC guardian :D!

**Pull request merged**